tests/virt_autotest/guest_installation_run.pm

```diff
@@ -16,7 +16,8 @@ use virt_utils;
 sub get_script_run {
     my $pre_test_cmd = "";
     if (is_s390x) {
-        $pre_test_cmd = "/usr/share/qa/tools/test_virtualization-virt_install_withopt-run";
+        $pre_test_cmd = "set -o pipefail;";
```

Comment:
Why do you need ```set -o pipefail```.

Reply:
Trigger CMD as `"/usr/share/qa/tools/test_virtualization-virt_install_withopt-run -f sles-15-sp6-64 -n 2 -r  2>&1 | tee /tmp/s390x_guest_install_test.log" ` run with ` /usr/share/qa/tools/test_virtualization-virt_install_withopt-run` and `create /tmp/s390x_guest_install_test.log `together in the same time. So, need to use with `set -o pipefail`